pubspec.yaml

```
@@ -80,13 +80,22 @@ flutter:
       - asset: assets/fonts/Pretendard-1.3.9/public/static/Pretendard-Black.otf
         weight: 900
     
+  # To add assets to your application, add an assets section, like this:
   assets:
     - assets/images/logo.svg
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)